lib/crowbar/init/application.rb

```diff
@@ -14,6 +14,8 @@
 # limitations under the License.
 #
 
+require "/opt/dell/crowbar_framework/lib/crowbar/upgrade_status"
```

Comment:
> Nope, that's not how we do imports :-)

:-(

please check again. we are in `/usr/share/crowbar-init/lib/crowbar/init`. how is this supposed to work there?